sdks/python/test-suites/gradle.properties

```diff
@@ -0,0 +1,38 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# Following properties define which Python versions be used when the tests running.
+# Versions must be indicated major and minor versions connected with period.
+# And if you want to use multiple versions, concat them with comma and no whitespace.
+
+# dataflow test-suites
+dataflow_precommit_it_task_py_versions=2.7,3.7
+dataflow_mongodbio_it_task_py_versions=3.5
```

Comment:
I don't think we have a dataflow MongoDB suite - I only see a direct runner suite.

Reply:
dataflow MongoDB suite is used here:
https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_PostCommit_Python_MongoDBIO_Load_Test.groovy#L54

Reply:
I see, thanks for correction. Then we could keep this setting, but, modify add it together with the Groovy file.